pipelines/simple-notebook-pipeline/README.md

```diff
@@ -0,0 +1,13 @@
+# Simple Notebook Pipeline on GCP
+This notebook shows how to compile and run a simple Kubeflow pipeline using Jupyter notebooks and Google Cloud Storage.  The pipeline is very simple, and is a helpful starting point for people new to Kubeflow.
+
+## Setup
+
+### Setup notebook server
+This pipeline requires you to [setup a notebook server](https://www.kubeflow.org/docs/notebooks/setup/) in the Kubeflow UI.  After you are setup, upload this notebook and then run it in the notebook server.

```

Comment:
Include a link to the notebook or instructions to use a Terminal within Jupyterhub to clone the repo.

Reply:
Added instructions in the README for cloning repo into notebook server.